packages/swing-store/src/swingStore.js

```diff
@@ -17,10 +18,20 @@ export const DEFAULT_LMDB_MAP_SIZE = 2 * 1024 * 1024 * 1024;
 export { makeSnapStore };
 
 export function makeSnapStoreIO() {
+  const { now } = performance;
+  // TODO: find a home for `export const makeMeasureSeconds = now => {...}`.
+  const measureSeconds = async fn => {
+    const t0 = now();
+    const result = await fn();
+    const durationMillisec = now() - t0;
+    return { result, duration: durationMillisec / 1000 };
+  };
```

Comment:
Thoughts on the best location for a function that accepts a `now` and returns a function that uses it to report the time taken to invoke a provided function? Is [telemetry](https://github.com/Agoric/agoric-sdk/tree/HEAD/packages/telemetry) a good fit, or should I just go straight to [internal](https://github.com/Agoric/agoric-sdk/tree/HEAD/packages/internal)?

Reply:
I don't think telemetry is the right home, probably the new internal? I'd want to have it be a maker that takes a `now` function.

Reply:
Done.